File: src/main/java/com/wgzhao/addax/admin/dto/AddaxReportDto.java
package com.wgzhao.addax.admin.dto;

import lombok.Data;

@Data
public class AddaxReportDto {
    private String jobName;
    private long startTimeStamp;
    private long endTimeStamp;
    private long totalCosts;
    private long totalBytes;
    private long byteSpeedPerSecond;
    private long recordSpeedPerSecond;
    private long totalReadRecords;
    private long totalErrorRecords;

    @Override
    public String toString() {
        return "AddaxReportDto{" +
                "jobName='" + jobName + '\'' +
                ", startTimeStamp=" + startTimeStamp +
                ", endTimeStamp=" + endTimeStamp +
                ", totalCoasts=" + totalCosts +
                ", totalBytes=" + totalBytes +
                ", byteSpeedPerSecond=" + byteSpeedPerSecond +
                ", recordSpeedPerSecond=" + recordSpeedPerSecond +
                ", totalReadRecords=" + totalReadRecords +
                ", totalErrorRecords=" + totalErrorRecords +
                '}';
    }
}

File: src/main/kotlin/com/wgzhao/addax/admin/dto/AddaxReportDto.kt
package com.wgzhao.addax.admin.dto

data class AddaxReportDto(
    val jobName: String,
    val startTimeStamp: Long = 0,
    val endTimeStamp: Long = 0,
    val totalCosts: Long = 0,
    val totalBytes: Long = 0,
    val byteSpeedPerSecond: Long = 0,
    val recordSpeedPerSecond: Long = 0,
    val totalReadRecords: Long = 0,
    val totalErrorRecords: Long = 0
)
